crates/flox/src/commands/environment.rs

```diff
@@ -129,6 +132,49 @@ impl EnvironmentCommands {
                     .await?
             },
 
+            EnvironmentCommands::WipeHistory {
+                // TODO use environment_args.system?
+                environment_args: _,
+                environment,
+            } => {
+                let environment_name = environment.as_ref().map(|e| e.to_str().unwrap());
+                let environment_ref: environment_ref::EnvironmentRef =
+                    resolve_environment_ref::<GitCommandProvider>(
+                        &flox,
+                        "wipe-history",
+                        environment_name,
+                    )
+                    .await?;
+
+                let environment = environment_ref
+                    .to_env::<GitCommandProvider, NixCommandLine>(&flox)
+                    .await
+                    .context("Environment not found")?;
+
+                match environment {
+                    CommonEnvironment::Named(env) => {
+                        if env.delete_symlinks().await? {
+                            let nix = flox.nix::<NixCommandLine>(vec![
```

Comment:
```suggestion
                            // The flox nix instance is created with `--quiet --quiet`
                            // because nix logs are passed to stderr unfiltered.
                            // nix store gc logs are more useful,
                            // thus we use 3 `--verbose` to have them appear.
                            let nix = flox.nix::<NixCommandLine>(vec![
```

we should probably make `Flox::nix` more configurable or pass quiet flags per command..